jcl/src/openj9.dtfj/share/classes/com/ibm/dtfj/javacore/parser/j9/section/thread/ThreadSectionParser.java

```diff
@@ -140,20 +138,35 @@ protected void threadInfo(boolean buildModel, boolean currentThread) throws Pars
 	 */
 	protected int processThreadandStackTrace(IAttributeValueMap javaThreadResults, boolean buildModel, boolean currentThread, int currentLineNumber) throws ParserException {
 		JavaThread javaThread = null;
+		// 3XMJAVALTHREAD            (java/lang/Thread getId:0xD, isDaemon:true)
+		processTagLineOptional(T_3XMJAVALTHREAD);
+		// 3XMJAVALTHRCCL            jdk/internal/loader/ClassLoaders$AppClassLoader(0x000000072254DA00)
+		IAttributeValueMap results = processTagLineOptional(T_3XMJAVALTHRCCL);
+		if (results != null) {
+			String cl_name = results.getTokenValue(IThreadTypes.CONTEXT_CLASSLOADER_OBJECT_FULL_JAVA_NAME);
+			long cl_address = results.getLongValue(IThreadTypes.CONTEXT_CLASSLOADER_OBJECT_ADDRESS);
```

Comment:
Please follow the existing naming convention, i.e. these locals should be `clName` and `clAddress`.

Reply:
I shall fix this. (I probably copied some of this from ClassLoaderSectionParser.java)